samples/esb/esb_monitor/Kconfig

```
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Enhanced ShockBurst: Monitor"
+
+config ESB_MONITOR_APP_LOG_LEVEL
+	int "Log level for the ESB MONITOR sample"
+	default 4
```

Please align this sample to current style of LOG Kconfig entry. Ie in samples/bluetooth/direct_test_mode/Kconfig.
```suggestion
module = ESB_MONITOR_APP
module-str = "ESB Monitor app"
source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
```